### PR TITLE
Add test for coverage

### PR DESCRIPTION
**Pull request by matsubara0507**
This PR improve line coverage of cmath.rb from 74.4% to 100%

### CODE REVIEW

File: test/test_cmath.rb [hunk 1]
```diff
@@ -3,7 +3,16 @@
 require 'cmath'
 
 class TestCMath < Test::Unit::TestCase
+  def test_any_method!
```
Comment (hsbt) [inline test/test_cmath.rb]: I'm okay with this test-case. But I hope to understand reason why the name of this test-case is `any_method!`. Can you elaborate on it?

Comment (matsubara0507) [reply test/test_cmath.rb]: Sory. I given not good name to this method. 
Rename test that check deprecated method.

Comment (matsubara0507) [reply test/test_cmath.rb]: Rename to `test_deprecated_method`, thanks.

Comment (hsbt) [reply test/test_cmath.rb]: I see. Thanks!

**Comment by hsbt** [comment]
Thanks a lot!

File: test/test_cmath.rb [hunk 2]
```diff
@@ -3,7 +3,16 @@
 require 'cmath'
 
 class TestCMath < Test::Unit::TestCase
+  def test_deprecated_method
+    $VERBOSE = true
```
Comment (znz) [inline test/test_cmath.rb:7]: If you change global state, it should restore in ensure clause.

**Comment by hsbt** [comment]
@znz Fixed at https://github.com/ruby/cmath/commit/460f5ff4a2ae568a5d55c86443894ef186ea6929

**Pull request merged**